WireUI/Sources/WireIndividualToTeamMigrationUI/Views/SelfProfileViewCallToActionBanner.swift

```diff
@@ -0,0 +1,103 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import UIKit
+import WireDesign
+import WireFoundation
+import WireReusableUIComponents
+
+public struct SelfProfileViewCallToActionBanner: View {
+
+    public enum Action: Sendable {
+        case createWireTeam
+    }
+
+    let actionCallback: @Sendable (Action) -> Void
+
+    public init(actionCallback: @escaping @Sendable (Action) -> Void) {
+        self.actionCallback = actionCallback
+    }
+
+    public var body: some View {
+        // if the device screen width is less than 400 (iPhone mini is 375), use a smaller padding
```

Comment:
Is this a todo?

Reply:
No, a leftover comment. The issue has been solved another way.